test/inductor/test_torchinductor.py

```diff
@@ -134,6 +134,11 @@
 i64 = torch.int64
 i32 = torch.int32
 
+def expectedFailureCodegenDynamicIfNotSM80OrLater(fn):
+    if not SM80OrLater:
+        fn._expected_failure_codegen_dynamic = True
```

Comment:
Would it be possible to pattern match the implementation of e.g., `dtypesIfCUDA` since this sounds like "expected failure codegen dynamic if cuda" https://github.com/pytorch/pytorch/blob/e00282fecfcb53790aebfb24cc48a8703577778e/torch/testing/_internal/common_device_type.py#L1278

Reply:
Yep, let's try something alike.